ETL/FactPatientMetrics/LoadFactPatientMetrics.sql

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Have you tried putting this ETL logic and all of the other ETLs  in a PROCEDURE? it is pretty common to have ETLs in PROCEDURES, if you don't have an ETL Tool sometimes a PROCEDURE and a scheduler is what works